cmd/snap/cmd_snap_op.go

```diff
@@ -183,24 +183,51 @@ type cmdRefresh struct {
 	Channel    string `long:"channel" description:"Refresh to the latest on this channel, and track this channel henceforth"`
 	Positional struct {
 		Snap string `positional-arg-name:"<snap>"`
-	} `positional-args:"yes" required:"yes"`
+	} `positional-args:"yes"`
 }
 
-func (x *cmdRefresh) Execute([]string) error {
+func refreshAll() error {
 	cli := Client()
-	name := x.Positional.Snap
-	opts := &client.SnapOptions{Channel: x.Channel}
-	changeID, err := cli.Refresh(name, opts)
+	updates, err := cli.List(&client.ListOptions{RefreshOnly: true})
```

Comment:
why are we implementing this in the client? wouldn't it be better to have it in snapd?